docs/content/en/docs/topics/provision-a-cluster/_index.md

```diff
@@ -14,123 +20,176 @@ description: Steps to provision a Cassandra cluster in Kubernetes
 * A Kubernetes environment
 * K8ssandra installed and running in Kubernetes - see [Quick start]({{< ref "getting-started" >}})
 
-## Steps
-
-### Use helm to get the running configuration
-
-For many basic configuration options, you may change values in the deployed YAML files. For example, you can scale up or scale down, as needed, by updating the YAML via `helm` command `--set` parameters.
+## Create a cluster
 
-Let's check the currently running values. First get the list of the installed K8ssandra chart. In this example, assume the `releaseName` was defined as `k8ssandra` on the `helm install` command.
+Suppose we install K8ssandra as follows:
 
 ```bash
-helm list
+helm install my-k8ssandra k8ssandra/k8ssandra -f k8ssandra-values.yaml
 ```
 
-**Output**:
+Assume that `k8ssandra-values.yaml` has these properties:
 
-```bash
-NAME     	  NAMESPACE	 REVISION   UPDATED                               STATUS  	CHART      APP VERSION
-k8ssandra	  default  	 1          2021-03-04 20:49:32.975090399 +0000   UTC	      deployed	 k8ssandra-1.0.0	              
+```yaml
+cassandra:
+  clusterName: my-k8ssandra
+  datacenters:
+  - name: dc1
+    size: 3
 ```
 
-You can specify the name of the installed cluster's `releaseName` to get the full manifest. 
+The helm instal command will result in the creation of a `CassandraDatacenter` object with the size set to 3. The cass-operator product that's installed by K8ssandra will in turn create the underlying StatefulSet that has 3 Cassandra pods.
```

Comment:
`product` feels out of place here, especially since we are dealing with OSS projects. I would replace it with `Deployment` since that is the actual Kubernetes object type that is installed for cass-operator.

Reply:
I agree - inc'd your comments with subsequent commit.